nextcord/state.py

```diff
@@ -2307,13 +2307,15 @@ def parse_auto_moderation_action_execution(self, data) -> None:
 
     def parse_guild_audit_log_entry_create(self, data) -> None:
         guild = self._get_guild(int(data["guild_id"]))
-        user_id = int(data["user_id"])
+        user_id = None if data.get("user_id") is None else int(data["user_id"])
         user = self.get_user(user_id)
+        if user_id is None or user is None:
```

Comment:
Little note on my changes here. The `users` parameter is used as a dictionary to resolve users found inside the entry. I am not sure why it does not use `get_user` in there, but thats irrelevant.

The ternary above is shorter than what you would think would be used.
```py
user_id = None if data.get("user_id") is None else int(data["user_id"])
user_id = int(data["user_id"]) if data.get("user_id") is not None else None
```